hoodie-client/src/main/java/com/uber/hoodie/index/hbase/HBaseIndex.java

```diff
@@ -194,6 +261,10 @@ private Connection getHBaseConnection() {
               if (!writeStatus.isErrored(rec.getKey())) {
                 java.util.Optional<HoodieRecordLocation> loc = rec.getNewLocation();
                 if (loc.isPresent()) {
+                  if (rec.getCurrentLocation() != null) {
+                    // This is an update, no need to update index
```

Comment:
check indentation..

Reply:
looks fine to me, do you see anything wrong ?

Reply:
ok I think I saw folded lines above and concluded this. looks good. my bad